mentoring/mentoring.py

```diff
@@ -235,6 +279,44 @@ def next_step_url(self):
         return '/jump_to_id/{}'.format(self.next_step)
 
     @XBlock.json_handler
+    def get_results(self, queries, suffix=''):
+        results = []
+        if not self.show_extended_feedback():
+            return {
+                'results': [],
+                'error': 'Results cannot be obtained.'
+            }
+        completed = True
+        choices = dict(self.student_results)
+        step = self.step
+        # Only one child should ever be of concern with this method.
+        for child in self.get_children_objects():
+            if child.name and child.name in queries:
+                results = [child.name, child.get_results(choices[child.name])]
+                completed = choices[child.name]['status']
+                break
+
+        message = self.get_message(True)
```

Comment:
@e-kolpakov No. In fact, I should not be checking for completion here at all, since it's a foregone conclusion that we'll be completed by this point, so I'm removing that check instead.